specs/agents/metadata.md

```diff
@@ -70,7 +70,7 @@ For official Elastic agents, the agent name should just be the name of the langu
 
 ### Cloud Provider Metadata
 
-[Cloud provider metadata](https://github.com/elastic/apm-server/blob/master/docs/spec/cloud.json)
+[Cloud provider metadata](https://github.com/elastic/apm-server/blob/master/docs/spec/v2/metadata.json#L5-L129)
```

Comment:
The exact line numbers are likely going to change. We could add a permalink to that revision but then it would link to a possibly outdated version. Although it's not a bummer if linking to out-of-sync line numbers, I think we can also just link to the file. The cloud section is not that hard to find.
```suggestion
[Cloud provider metadata](https://github.com/elastic/apm-server/blob/master/docs/spec/v2/metadata.json)
```